lib/kredis/types/counter.rb

```diff
@@ -1,19 +1,19 @@
 class Kredis::Types::Counter < Kredis::Types::Proxying
-  proxying :multi, :set, :incrby, :decrby, :get, :del, :exists?
+  proxying :multi, :set, :get, :del, :exists?
```

Comment:
Removing these two from the proxy would be a breaking API change. Potentially someone were calling these directly. Think we need to leave them exposed.